content/docs/tasks/traffic-management/egress/egress-kubernetes-services/index.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Kubernetes Services for Egress Traffic
+description: Shows how to configure Istio  Kubernetes External Services.
+keywords: [traffic-management,egress]
+weight: 60
+---
+
+Sometimes, for various reasons, for example in pre-Istio scenarios,
```

Comment:
Please be clear and explicit about the scenarios where the k8s `ExternalName` or `Endpoints are used to access external services. As a reader, I need to be sure that this is the task I need before I begin going through the steps.